project/ScoverageSettings.scala

```diff
@@ -0,0 +1,27 @@
+import sbt.Def
+import scoverage.ScoverageKeys
+
+object ScoverageSettings {
+  lazy val scoverageSettings: Seq[Def.SettingsDefinition] = {
+    val excludedFiles = Seq(
+      """<empty>""",
+      """Reverse.*""",
```

Comment:
would this not be covered already if we exclude the `*.Reverse.*` package below?

Reply:
Good point, I'll check